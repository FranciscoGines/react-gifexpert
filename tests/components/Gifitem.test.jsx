import { render } from "@testing-library/react";
import { GifItem } from "../../src/components/Gifitem";


describe('Pruebas en el <Gifitem />', () => { 

    const title = 'Gines';
    const url   = 'https://one-punch.com/gines.jpg';
    

    test('debe de hacer march con el snapshot', () => { 
        
       const { container } = render( <GifItem title={ title } url={ url } /> )
        expect ( container ).toMatchSnapshot();
    });
});