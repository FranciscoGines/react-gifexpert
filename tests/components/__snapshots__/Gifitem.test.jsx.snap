// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el <Gifitem /> debe de hacer march con el snapshot 1`] = `
<div>
  <div
    class="card"
  >
    <img
      alt="Gines"
      src="https://one-punch.com/gines.jpg"
    />
    <p>
      Gines
    </p>
  </div>
</div>
`;
